### PR TITLE
feat: Android release structure to maven central

### PR DESCRIPTION
**Pull request by JamieSinn**
wip

### CODE REVIEW

File: kotlin-example/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,7 +3,3 @@ distributionPath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-
```
Comment (kaushalkapasi) [inline kotlin-example/gradle/wrapper/gradle-wrapper.properties:6]: why removed?

Comment (JamieSinn) [reply kotlin-example/gradle/wrapper/gradle-wrapper.properties:6]: This was part of the wrapper autogenerated changes.

Comment (kaushalkapasi) [reply kotlin-example/gradle/wrapper/gradle-wrapper.properties:6]: add it back please.. we use this for minification testing & validation on the kotlin-example app

Comment (JamieSinn) [reply kotlin-example/gradle/wrapper/gradle-wrapper.properties:6]: Moved to the proper spot in gradle.properties.

R8 is now enabled by default for both app and Android library projects using Android Gradle plugin 3.4.0 and higher.

**Review by kaushalkapasi — APPROVED** [review]

**Pull request merged**